src/app/components/Table/Table.tsx

```diff
@@ -80,15 +80,18 @@ export const Table: FunctionComponent<TableProps> = ({
       ouiaId={ariaLabel}
     >
       <Thead>
-        <Tr>
+        <Tr ouiaId="table-head">
           {transformColumns(columns).map((column) => (
             <Th key={column}>{column}</Th>
           ))}
         </Tr>
       </Thead>
       <Tbody>
         {transformRows(rows, columns).map((row: TableRow, rowIndex) => (
-          <Tr key={(row.originalData?.id as string) ?? rowIndex}>
+          <Tr
+            ouiaId={row.originalData?.name as string}
```

Comment:
```suggestion
            ouiaId=`{row.originalData?.name as string}-processor`
```

Reply:
Maybe would be good to suffix with processor? I mean, in mock data, we put 'processor' into each processor name, however real user do not need to do it.

Reply:
@jomarko currently we use "Processor one" and "Processor two". So your suggestion is look like "Processor one-processor" and "Processor two-processor". I think that it looks wierd in a test. The other opportunity is a replacement of the name by the processor id. Something like "processor-ab65ec62-1f23-4dd7-b106-e4158baf8228".

In case of e2e tests the id is better option (see https://github.com/kiegroup/kogito-apps/blob/main/ui-packages/packages/trusty/cypress/e2e/TrafficViolation.ts#L51-L54).

Reply:
The second reason why do not use `processor` tag is that the component shows diferent objects like instances, etc.